packages/dumb/src/components/settings/history/history-table.tsx

```diff
@@ -140,34 +132,7 @@ export const HistoryParameterTable: FunctionComponent<HistoryParameterTableProps
     }))
   }
 
-  const renderSpannedRow = (normalizedColumns: NormalizedColumn[], rowKey: number, rowData: HistorizedRow): JSX.Element => {
-    const { onSwitchToDaily } = props
-    let content = rowData.spannedContent
-    if (!content) {
-      content = '&nbsp;'
-    }
-    const handleSwitchToDaily = (): void => {
-      onSwitchToDaily(rowData.mLatestDate, 'Diabeloop parameters history')
-    }
-    const dateString = rowData.mLatestDate.toISOString()
-    return (
-      <tr id={`parameters-history-${dateString}`} key={rowKey} className={styles.spannedRow}>
-        <td colSpan={normalizedColumns.length}>
-          {content}
-          <i
-            id={`parameters-history-${dateString}-link-daily`}
-            role="button"
-            tabIndex={0}
-            data-date={dateString}
-            className={`icon-chart-line ${styles.clickableIcon} parameters-history-link-daily`}
-            onClick={handleSwitchToDaily}
-            onKeyPress={handleSwitchToDaily}
-          />
-        </td>
-      </tr>)
-  }
-
-  const renderRow = (normalizedColumns: NormalizedColumn[], rowKey: number, rowData: HistorizedRow, /** @type {string} */ trClassName = undefined): JSX.Element => {
+  const renderRow = (normalizedColumns: NormalizedColumn[], rowKey: number, rowData: HistorizedParameter, /** @type {string} */ trClassName = undefined): JSX.Element => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏We usually would write this this way:

```suggestion
  const renderRow = (normalizedColumns: NormalizedColumn[], rowKey: number, rowData: HistorizedParameter, trClassName: string | undefined = undefined): JSX.Element => {
```
I cannot say if your solution is better or worse though. :wink:

Reply:
me neither, i will change it the way you propose